docs/06-contributors/999-rfcs/000-stories/story-13.md

```diff
@@ -113,14 +92,14 @@ resource TaskList {
     let output = MutArray<str>[];
     for id in task_ids {
       let j = this.get_task(id); 
-      let title = str.from_json(j.get("title"));
+      let title = j.get_str("title"); // notice this require us to create a small "get_str" function  implementation that should be removed by next sprint, maybe there is a different way around this. @hasanaburayyan and @staycoolcall911 ?
```

Comment:
I'm wondering if we could add a generic `get()` instead of `get_str()`. If we could do that, we can probably keep it and not remove it in a later sprint.